index.js

```diff
@@ -24,7 +24,10 @@ exports.debug_mode = /\bredis\b/i.test(process.env.NODE_DEBUG);
 
 // hiredis might not be installed
 try {
-    parsers.push(require("./lib/parsers/hiredis"));
+    // Confuse bundlers so that they won't try
+    // to include this native module.
+    var hiredisParserPath = "./lib/parsers/hiredis";
+    parsers.push(require(hiredisParserPath));
```

Comment:
You're right. That's likely. I guess the best way would be what I'm doing right now, exclude hiredis from require resolution. Thanks for your great code anyway ^^